src/omotes_simulator_core/entities/assets/ates_cluster.py

```diff
@@ -209,7 +208,7 @@ def _init_rosim(self) -> None:
         """Function to initailized Rosim from XML file."""
         path = os.path.dirname(__file__)
         xmlfile = os.path.join(path, "bin/sequentialTemplate_v0.4.2_template.xml")
-        with open(xmlfile, "r") as fd:
+        with open(xmlfile) as fd:
```

Comment:
This was an automatic fix by the linter (ruff), see https://docs.astral.sh/ruff/rules/redundant-open-modes/ 

However, I understand the reasoning.  The default is read-only,  the lowest access-mode available.  If you only need read-mode you don't really care what mode it's actually opened in,  as long as the open-call succeeds. If you don't care, why specify the parameter. 

Also,  programmers should be aware of the defaults used by the functions they're calling.  The default is read-only, so why would you repeat yourself by specifying the parameter.  There are 6 additional parameters for `open()` that we also don't specify here, because the defaults are fine.